distributions/distribution-list.md

```diff
@@ -1,10 +1,10 @@
 # Distributions
 
-Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have also created alternative distributions of Chef's Open Source products. These are listed below.
+Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have created alternative distributions of Chef's Open Source products. These are listed below.
 
-## cc-build
+## Cinc
 
-This includes community distributions of Chef's software (including Chef Infra, InSpec, Habitat, and Automate).
+The Cinc project currently includes community distributions of Chef Infra-client (Cinc Client) and Chef InSpec (Cinc Auditor).
 
```

Comment:
Could you please add our website as well? https://cinc.sh

Reply:
done!